### ISSUE TITLE
The contrast loss

### ISSUE THREAD

**Issue by amadeusuzx** [HarryVolek/PyTorch_Speaker_Verification#70]
Hi there,
Maybe I just didn't see it, but there seems no scripts implementing the contrast loss, which takes into account of the most aggressive pairs in the similar matrix?

**Comment by amadeusuzx** [comment]
Sorry, I think I understand now -- TI-SV uses softmax loss .
